DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/MappingConfig.cs

```diff
@@ -0,0 +1,17 @@
+﻿using GitHubExtension.Notes.DAL.Model;
+using GitHubExtension.Notes.WebApi.ViewModels;
+
+namespace GitHubExtension.Notes.WebApi
+{
+    public static class MappingConfig
+    {
+        public static void RegisterMaps()
+        {
+            AutoMapper.Mapper.Initialize(config =>
```

Comment:
That was convenient for me to use it in my module, but team decided to stay on a manual mappers as extension methods. Added new commit with manual mapper instead of AutoMapper.